### ISSUE TITLE
why "xy_loss" have the factor of "box_loss_scale"?

### ISSUE THREAD

**Issue by peteryvqi** [zzh8829/yolov3-tf2#213]
models->YoloLoss
....
def yolo_loss:
box_loss_scale = 2 - true_wh[..., 0] * true_wh[..., 1]
# 5. calculate all losses
        xy_loss = obj_mask * box_loss_scale * \
            tf.reduce_sum(tf.square(true_xy - pred_xy), axis=-1)
        wh_loss = obj_mask * box_loss_scale * \
            tf.reduce_sum(tf.square(true_wh - pred_wh), axis=-1)
......
I can understand that "wh_loss" contain  "box_loss_scale"。
but ,why "xy_loss" have "box_loss_scale" the same? I don't  know.
Thanks

**Comment by abhishek-verma-github** [comment]
box_loss_scale is actually a way to give high priority to small boxes. In YOLOv1 papers, Joesph used a 'Lambda_coordinate' a multiplicative factor to be multiply while calculating xy_loss to emphasize network to focus on learning to predict xy coordinates more than classification.
